modules/sugar/src/main/ts/ephox/sugar/api/Main.ts

```diff
@@ -19,6 +19,7 @@ import * as Body from './node/Body';
 import * as Comment from './node/Comment';
 import * as Comments from './node/Comments';
 import Element from './node/Element';
+import * as ElementInstances from './node/ElementInstances';
```

Comment:
Mind adding sugar to `versions.txt` so we know to give it a minor version bump?

Reply:
Oh sorry about that!
Fixed.